src/features/todos.ts

```diff
@@ -1,8 +1,12 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction, Slice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
-export const todosSlice = createSlice({
+export const todosSlice: Slice<Todo[]> = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    setTodos(todos, action: PayloadAction<Todo[]>) {
+      todos.push(...action.payload);
```

Comment:
Using `push` here mutates the state directly, which is not recommended. Instead, return a new array with the todos, such as `return [...action.payload];` to maintain immutability.